.github/actions/install-flux/action.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Install Flux CLI and Source Controller"
+description: "Installs Flux CLI and Source Controller."
+inputs:
+  flux-version:
+    description: "Flux version"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install Flux
+      id: export-token
+      shell: bash
+      env:
+        GITEA_PASSWORD: ${{ env.GITEA_PASSWORD }}
```

Comment:
Is this env var used? If so, can you add a comment explaining why it's necessary?

Reply:
copy paste error - should be fixed now